modules/ztp-precaching-ztp-config.adoc

```diff
@@ -75,6 +75,8 @@ spec:
             - name: "ens1f0"
               macAddress: "AA:BB:CC:11:22:33"
 ----
+<1> References the cluster image set that is used for deployment, unless you specify a different image set in the `spec.clusters.clusterImageSetNameRef` field.
```

Comment:
```suggestion
<1> Specifies the cluster image set used for deployment, unless you specify a different image set in the `spec.clusters.clusterImageSetNameRef` field.
```
- Would `specifies` be more accurate than `references` here?
- Edit for minimalism